tools/build.csx

```diff
@@ -88,53 +83,6 @@ targets.Add("build", DependsOn("clean", "restore", "versionInfoFile"), () => Run
 
 targets.Add("versionInfoFile", () => Cmd(gitversion, $"/updateAssemblyInfo {versionInfoFile} /ensureAssemblyInfo"));
 
-targets.Add(
-    "coverity",
-    DependsOn("clean", "coverityDirectory", "restore"),
-    () =>
-    {
-        Cmd(
-            "cov-build",
-            $@"--dir {coverityResultsDirectory} ""{msBuild}"" {solution} /target:Build /p:configuration=Release /nr:false /verbosity:minimal /nologo /fl /flp:LogFile=artifacts/logs/Coverity-Build.log;Verbosity=Detailed;PerformanceSummary");
-
-        var version = ReadCmdOutput(".", gitversion, "/showvariable SemVer");
-        var coverityToken = Environment.GetEnvironmentVariable("COVERITY_TOKEN");
-        var coverityEmail = Environment.GetEnvironmentVariable("COVERITY_EMAIL");
-        var repoCommitId = Environment.GetEnvironmentVariable("APPVEYOR_REPO_COMMIT");
-
-        var coverityZipFile = coverityDirectory + "/coverity.zip";
-        Cmd("7z", $"a -r {coverityZipFile} {coverityResultsDirectory}");
-
-        using (var client = new HttpClient())

```

Comment:
Now we don't need the System.Net.Http lines above.

Reply:
Ah, I was counting on VSCode to tell me this kind of thing, but it didn't... I'll remove the unnecessary usings.